xml/gl.xml

```diff
@@ -10034,24 +10037,28 @@ typedef unsigned int GLhandleARB;
             <param group="DrawBufferName"><ptype>GLint</ptype> <name>drawbuffer</name></param>
             <param><ptype>GLfloat</ptype> <name>depth</name></param>
             <param><ptype>GLint</ptype> <name>stencil</name></param>
+            <glx type="render" opcode="360"/>
```

Comment:
In OpenGL3_0.txt, opcode 360 was assigned to glClearBufferfi, which I don't see included here.

Reply:
```xml
<command>
    <proto>void <name>glClearBufferfi</name></proto>
    <param group="Buffer"><ptype>GLenum</ptype> <name>buffer</name></param>
    <param group="DrawBufferName"><ptype>GLint</ptype> <name>drawbuffer</name></param>
    <param><ptype>GLfloat</ptype> <name>depth</name></param>
    <param><ptype>GLint</ptype> <name>stencil</name></param>
    <glx type="render" opcode="360"/>
</command>
```